samples/rag-aisearch/README.md

```diff
@@ -13,19 +13,55 @@ The sample is available in the following language stacks:
 
 Please refer to the [root README](../../README.md#requirements) for common prerequisites that apply to all samples.
 
-## Running the sample
-
-This sample requires creating an Azure AI Search with Semantic Ranker. You can do this by following the [Azure AI Search quickstart](https://learn.microsoft.com/en-us/azure/search/search-create-service-portal)
+This sample requires creating an Azure AI Search with (optional) Semantic Ranker. You can do this by following the [Azure AI Search quickstart](https://learn.microsoft.com/en-us/azure/search/search-create-service-portal)
 and optionally [enable semantic ranking](https://learn.microsoft.com/en-us/azure/search/semantic-how-to-enable-disable?tabs=enable-portal) for semantic ranking and captions features.
 
+## Managed Identity and Endpoint Configuration
+
+1. Configure AI Search API Access Control (Strongly Recommended to use RBAC) -
```

Comment:
nit: the use of capitalization for "Strongly Recommended" here and below is a little weird. I suggest using normal sentence case.